elements/lisk-dpos/src/dpos.ts

```diff
@@ -70,6 +73,9 @@ export class Dpos {
 
 	public constructor({
 		chain,
+		initDelegates,
+		genesisHeight = DEFAULT_GENESIS_HEIGHT,
+		initRound = DEFAULT_INIT_ROUND,
```

Comment:
Same as above. These should not have default values in `dpos`. Having such default values on once place will make it easy to maintain.